projects/1.20-fabric/assets/modrinth-big-globe/bigglobe/lang/zh_cn.json

```
@@ -0,0 +1,309 @@
+{
+	"generator.bigglobe.bigglobe": "Big Globe世界",
+	"generator.bigglobe.experiments": "Big Globe测试",
+
+	"block.bigglobe.ashen_netherrack": "积灰下界岩",
+	"block.bigglobe.blazing_blossom": "烈焰花",
+	"block.bigglobe.budding_quartz": "石英母岩",
+	"block.bigglobe.charred_button": "焦化木按钮",
+	"block.bigglobe.charred_door": "焦化木门",
+	"block.bigglobe.charred_fence": "焦化木栅栏",
+	"block.bigglobe.charred_fence_gate": "焦化木栅栏门",
+	"block.bigglobe.charred_grass": "焦化草",
+	"block.bigglobe.charred_hanging_sign": "焦化木悬挂式告示牌",
```

```suggestion
	"block.bigglobe.charred_hanging_sign": "悬挂式焦化木告示牌",
```
原版类物品变种请参照[mcw:MCW:ST](https://zh.minecraft.wiki/MCW:ST)的格式。